storage/postgres/queue_batching.go

```diff
@@ -0,0 +1,147 @@
+// +build batched_queue
```

Comment:
_Probably_ best to delete this file for now - it's build tagged so won't generally be used.

Reply:
It gets used in the testing.  If I remove it, then the unit tests fail.